library/Zend/Code/Reflection/ParameterReflection.php

```diff
@@ -98,11 +98,17 @@ public function getType()
         return;
     }
 
+    /**
+     * @return string
+     */
     public function toString()
     {
         return parent::__toString();
     }
 
+    /**
+     * @return string
+     */
     public function __toString()
```

Comment:
If this method only calls `parent::__toString()`, it has no use and can be removed.